src/inverseRobot.js

```diff
@@ -8,6 +8,20 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+  const robotValues = Object.values(robot);
+
+  for (const char of robotValues) {
+    if (robotValues.indexOf(char) !== robotValues.lastIndexOf(char)) {
+      return null;
+    }
+  }
```

Comment:
redundant code, you can to it more simple

Reply:
Yap. Thanks!